src/Model/Comment.php

```diff
@@ -104,6 +104,7 @@ class Comment extends DataObject
         'AuthorName' => 'Varchar',
         'RSSName' => 'Varchar',
         'DeleteLink' => 'Varchar',
+        'Date' => 'Varchar',
```

Comment:
Shouldn't it be a DateTime object :/ that way `.Nice`, etc will work in templates